patches/chrome-chrome_proxy-chrome_proxy_main_win.cc.patch

```diff
@@ -1,14 +1,13 @@
 diff --git a/chrome/chrome_proxy/chrome_proxy_main_win.cc b/chrome/chrome_proxy/chrome_proxy_main_win.cc
-index 6d79dd32ff183ce06a556b3e7ec60bd53ff4f917..2b316dd01dea0ed2b6703f037745da54645e3950 100644
+index 6d79dd32ff183ce06a556b3e7ec60bd53ff4f917..6cb9d7083d893677f11c1df8b501823e0638b6e6 100644
 --- a/chrome/chrome_proxy/chrome_proxy_main_win.cc
 +++ b/chrome/chrome_proxy/chrome_proxy_main_win.cc
-@@ -13,7 +13,11 @@
+@@ -13,7 +13,10 @@
  namespace {
  
  constexpr base::FilePath::CharType kChromeExecutable[] =
-+#if defined(BRAVE_CHROMIUM_BUILD)
 +    FILE_PATH_LITERAL("brave.exe");
-+#else
++#if 0
      FILE_PATH_LITERAL("chrome.exe");
 +#endif
```

Comment:
Wonder if we can simplify this by using something like `true ? FILE_PATH_LITERAL("brave.exe") :`

Reply:
That would certainly work too. I'll change it if that's preferred.

Reply:
Actually, I think I'll leave it as I had it for now if you don't mind, since doing what you suggest would go against some of the rules we have in the ["Patching Chromium" wiki page](https://github.com/brave/brave-browser/wiki/Patching-Chromium):

  _"If possible write the patch to add a new line vs appending/prepending to an existing line."_

I'm aware though that your suggestion means less lines added in the patch, but my hesitation comes from the fact that, if we're going to modify the existing line, we could very well then simply replace the string doing something like this:
```diff
- FILE_PATH_LITERAL("chrome.exe");
+ FILE_PATH_LITERAL("brave.exe");
```
WDYT?

Reply:
@mkarolin It doesn't seem this is blocking the merge, so I'm going to merge this PR today if it's ok to prevent it from getting conflicts (had to resolve some today already) if it's ok.